machines/README.md

```diff
@@ -0,0 +1,26 @@
+# Vagrantfiles for WSO2 Identity Server
```

Comment:
Shouldn't "Vagrantfiles" be singular as we only have one Vagrantfile?

Reply:
It should be Vagrantfile. Fixed it.